templates/ButtonBox.js

```diff
@@ -1,6 +1,7 @@
-import React, { PropTypes } from 'react'
+import React from 'react'
 import { TouchableOpacity, Text, Image } from 'react-native'
 import styles from './Styles/ButtonBoxStyles'
+import PropTypes from 'prop-types'
```

Comment:
Gonna need to add this package to the install script... it looks like this

```js
  // add prop types
  await ignite.addModule('prop-types', { version: '15.6.0' })
```

And would go along these:  https://github.com/infinitered/ignite-dev-screens/blob/master/plugin.js#L25

Reply:
I wondered about that.  Added it